azext_iot/central/_help.py

```diff
@@ -44,6 +57,158 @@ def load_central_help():
     _load_central_monitors_help()
     _load_central_command_help()
     _load_central_compute_device_key()
+    _load_central_export_help()
+
+
+def _load_central_export_help():
+    helps[
+        "iot central export"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoT Central exports.
```

Comment:
`Manage and configure IoT Central data exports.`?

Reply:
Fixed